modules/nf-core/pairtools/sample/main.nf

```diff
@@ -0,0 +1,38 @@
+process PAIRTOOLS_SAMPLE {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::pairtools=0.3.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pairtools:0.3.0--py37hb9c2fc3_5' :
+        'quay.io/biocontainers/pairtools:0.3.0--py37hb9c2fc3_5' }"
+
+    input:
+    tuple val(meta), path(input), val(fraction)
```

Comment:
Could you change the `input` variable name to something a bit more descriptive (like file type, file content...)